apps/studio/.storybook/main.ts

```diff
@@ -14,7 +14,7 @@ const config: StorybookConfig = {
   ],
```

Comment:
Stories load correctly

For last step which says "Chromatic snapshots are taken as expected", how to verify this?

Reply:
sorry autogenerated description via AI, but can still see in chromatic that the stories are rendered with `gsib` and `mobile` viewports when they are used. 

![Screenshot 2024-07-17 at 2.18.44 PM.png](https://graphite-user-uploaded-assets-prod.s3.amazonaws.com/0yF1kSri7ug0Z2JQvKtX/9e7aa614-0955-4d9e-89b5-60313e5474ef.png)